.github/workflows/ci.yml

```diff
@@ -0,0 +1,64 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
does that mean CI won't run on backporting PR to the release branches?

Reply:
I've fixed this. Thanks @mrueg !